site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-type.vm

```diff
@@ -193,6 +193,11 @@
                             #set($siteType = false)
                             #set($siteType = $!templateSite.type)
 
+                            #if (
+                                ($siteType == $projectSiteType && $canAddProject) || 
+                                ($courseSiteTypeStrings.contains($siteType) && $canAddCourse) ||
+                                ($siteType == $portfolioSiteType && $canAddPortfolio)
+                             )
                             #if($siteTypePrev != $siteType)
```

Comment:
I can't think of a clean way to do that. The second if statement determines whether the current template is from a previously unseen site type, and then builds the li element for that site type. It should end after the li block with class="siteTypeRow $siteType" ends (old line 210, new 215).

The if statement I added should contain that, but also the rendering of the individual li element for that template. That's the li block with id="rowTemplate$templateNum" (old line 222-229), and  #set($SiteTypePrev = $!templatesite.type) (old line 230). That individual li element has to be rendered even if the current template's site type has been seen before.